scripts/predict_retinotopy/README.md

```diff
@@ -2,13 +2,13 @@
 
 This directory contains the scripts necessary to produce the figures and results from the paper "Movies reveal the fine-grained organization of infant visual cortex" by Ellis, Yates, Arcaro, & Turk-Browne.  
 
-The primary script to refer to is `predict_retinotopy.ipynb`, a jupyter notebook that aggregates the data and outputs of other functions to produce the statistics and figures from the paper. To be able to run this script, you will need to have all of the data downloaded into the `data/` directory. Specifically, you need the [https://doi.org/10.5061/dryad.7h44j0ztm](retinotopy data) and the preprocessed movie data (Dryad link to come). 
+The primary script to refer to is `predict_retinotopy.ipynb`, a jupyter notebook that aggregates the data and outputs of other functions to produce the statistics and figures from the paper. To be able to run this script, you will need to have all of the data downloaded into the `data/` directory. Specifically, you need the [retinotopy data](https://doi.org/10.5061/dryad.7h44j0ztm) and the preprocessed movie data (Dryad link to come). 
 
 The only change to the scripts that you must make in order to run the code is the path specified in `utils.py`.
 
 The notebook should run without further edits; however, some cells in the notebook will take a long time to run as they are generating files. Ruunning the notebook will allow you to replicate the results from the paper, although some statistics might differ slightly based on random seeds.  
 
```

Comment:
Yeah it is true. I am not uploading lots of surfaces that are intermediate products of the analyses. They take up about 80G and are deterministic. Instead, I upload the original and the summary outputs. The notebook explains this point and how to avoid it.  Updated this section to clarify.